adafruit_cap1188/cap1188.py

```diff
@@ -171,6 +175,29 @@ def sensitivity(self, value):
         new_setting = self._read_register(_CAP1188_SENSITIVTY) & 0x8F | value
         self._write_register(_CAP1188_SENSITIVTY, new_setting)
 
+    @property
+    def averaging(self):
```

Comment:
Split the two methods you added into three separate pairs of functions, so 6 functions total. Have each set of functions handle one of the things you can set in register 0x24. That way the user won't have to pass all three values if they only want to set one. Basically, have the setter functions read the register, then only change the bits pertinent to that specific setting and then write to the register.

An alternative way to do this without creating any more functions would be to have the user pass up to 3 values to the setter function, making the default value of each None, and then reading from the register and only modifying the bits that the user wants to change. Personally, I prefer the first option, but I'd be okay with the second.